spec.md

```diff
@@ -768,7 +769,10 @@ For success responses, the following fields are defined:
 | Response Field | Type | Description |
 | --- | --- | --- |
 | state* | string | Valid values are `in progress`, `succeeded`, and `failed`. While `"state": "in progress"`, the Platform SHOULD continue polling. A response with `"state": "succeeded"` or `"state": "failed"` MUST cause the Platform to cease polling. |
+
 | description | string | A user-facing message that can be used to tell the user details about the status of the operation. |
+| status_code | number | The HTTP status code that would have been returned if the operation would have been executed synchronously. If the state is `failed` this field SHOULD be present and the value MUST be an integer in the range of 400 to 599. This field MUST NOT be present for any other state. |
```

Comment:
This is the line that I don't like: `The HTTP status code that would have been returned if the operation would have been executed synchronously.`. That suggests that only the HTTP codes defined for a synchronous response are permitted.

What about `An HTTP code that describes why the operation failed.`?

Reply:
Or, how about we explicitly call out the status codes that can be returned? This would help platforms build a better UX around this:
`200 OK` / `400 Bad Request` / `409 Conflict` seem appropriate from the Provision section.

Reply:
> That suggests that only the HTTP codes defined for a synchronous response are permitted.

The sync case's status codes can be anything, not just what's listed in the table - I don't think we need to list all possible values since we don't in the sync case. As for your proposed alternative text in the first comment, I'm ok with it but that's because I don't see the difference  :-)

Reply:
Thinking about this more... I do think it's important that the spec correlate the sync and async flows - meaning the `status_code` isn't just some random HTTP code - if the operation were invoked twice (once sync and once async) then typically people would see the same `status_code` in both cases, just in different locations/messages. W/o that I fear some people will not realize what we're trying to do and come up with different codes between the two flows.